commons/src/main/java/io/hyscale/commons/utils/OperatingSystemHelper.java

```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright 2019 Pramati Prism, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.hyscale.commons.utils;
+
+import org.apache.commons.lang3.StringUtils;
+
+import io.hyscale.commons.constants.ToolConstants;
+
+//
+public class OperatingSystemHelper {
```

Comment:
Rename something specific to windows as the state in the class FORWARD_SLASH is more specific to windows
Suggestion: WindowsQualifier / WindowsTransformer / WindowsManager

Reply:
Also this modification is very specific to installer.
If this should run with java -jar then modification should not happen that means this modification is triggered through installer with property **HYSCALE_HOST_FS** check